web/core/components/issues/issue-detail-widgets/attachments/quick-action-button.tsx

```diff
@@ -78,9 +78,15 @@ export const IssueAttachmentActionButton: FC<Props> = observer((props) => {
   });
 
   return (
-    <button {...getRootProps()} type="button" disabled={disabled}>
-      <input {...getInputProps()} />
-      {customButton ? customButton : <Plus className="h-4 w-4" />}
-    </button>
+    <div
```

Comment:
Why a wrapper div? can't we add thus directly to the button?

Reply:
I initially tried applying it directly to the button, but it was causing issues with the functionality. That's why I opted to wrap it in a div to maintain proper behaviour.